### DISCUSSION TITLE
posts/React/%EC%84%9C%EB%B2%84%EC%82%AC%EC%9D%B4%EB%93%9C%20%EB%A0%8C%EB%8D%94%EB%A7%81%EC%97%90%20%EB%8C%80%ED%95%B4%20%EC%95%8C%EC%95%84%EB%B3%B4%EC%9E%90

### DISCUSSION THREAD

**Discussion by giscus** [phnml1/my-blog#16] [Announcements]
# posts/React/%EC%84%9C%EB%B2%84%EC%82%AC%EC%9D%B4%EB%93%9C%20%EB%A0%8C%EB%8D%94%EB%A7%81%EC%97%90%20%EB%8C%80%ED%95%B4%20%EC%95%8C%EC%95%84%EB%B3%B4%EC%9E%90

클라이언트 사이드 렌더링(CSR)과 서버 사이드 렌더링(SSR)에 대해서 그리고 SSR을 활용한 next.js 예제와 SSG의 개념과 예제까지 알아보자.

https://phnml1.github.io/posts/React/%EC%84%9C%EB%B2%84%EC%82%AC%EC%9D%B4%EB%93%9C%20%EB%A0%8C%EB%8D%94%EB%A7%81%EC%97%90%20%EB%8C%80%ED%95%B4%20%EC%95%8C%EC%95%84%EB%B3%B4%EC%9E%90

**Comment by choihooo** [comment]
저도 next로 블로그 만들고 있는데 앱 라우터 써서 하고 있지만 유용하게 들은 거 같아요.

**Comment by jihye9549** [comment]
당장 next.js를 쓰고 싶게 만드는 필력이네요
